src/classes/CON_Email.cls

```diff
@@ -176,26 +178,29 @@ public class CON_Email {
 
             this.allFields = new List<CON_EmailField>();
             this.valuedFields = new List<CON_EmailField>();
-            Map<String, Schema.DescribeFieldResult> fieldMap = UTIL_Describe.getFieldsOfType('Contact', 'EMAIL');
-            this.emailLabel = fieldMap.get('Email').getLabel();
+
+            if (contactEmailFieldMap == null) {
+                contactEmailFieldMap = UTIL_Describe.getFieldsOfType('Contact', 'EMAIL');
```

Comment:
I think we are in the clear here but this may be the kind of thing that clues us into notifying QA to test in both namespaced and non-namespaced environments. Since this is a standard object and field, we're good, just want to point it out, more as a reference/reminder to myself.